build/common.sh

```diff
@@ -1148,6 +1151,9 @@ function kube::release::gcs::verify_release_files() {
 #   publish_file: the GCS location to look in
 # Returns:
 #   If new version is greater than the GCS version
+#
+# TODO(16529): This should all be outside of build an in release, and should be
+# refactored to reduce code duplication.
```

Comment:
I considered doing this work in this PR, but decided that I wanted to keep this as low-impact as possible, since it needs to be cherrypicked.  Once things stabilize in 1.1, #16529 can be targeted for 1.2.